client/src/test/java/com/aws/greengrass/cli/CLITest.java

```diff
@@ -65,6 +67,58 @@ protected NucleusAdapterIpc providesAdapter() {
                 .execute(args);
     }
 
+    @Test
+    void pubCommand() {
```

Comment:
if you don't specify either `--pubsub` or `--iotcore` what happens? That should be an error, we need exactly one of those to be specified.